apps/explorer/src/components/orders/BridgeDetailsTable/RefundStatus.tsx

```
@@ -0,0 +1,51 @@
+              contentsToDisplay={
+                <RefundAddressWrapper>
+                  <AddressLink address={refundWalletAddress} chainId={refundChainId} showNetworkName />
+                </RefundAddressWrapper>
+              }
+            />
+          </RefundStatusText>
+        )
+      }
+      return <RefundStatusText status={status}>Refund completed</RefundStatusText>
+
+    case RefundStatusEnum.NOT_INITIATED:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)